x-pack/test/performance/tests/playwright/login.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import Url from 'url';
+import { FtrProviderContext } from '../../ftr_provider_context';
+import { PROFILES } from '../user_action_profiles';
+
+export default function login({ getService }: FtrProviderContext) {
+  describe('login', () => {
+    const config = getService('config');
+    const playwright = getService('playwright');
+    const { step } = playwright.makePage('login', { autoLogin: false });
+
+    step('Go to Kibana login page', async ({ page }) => {
+      const kibanaUrl = Url.format({
+        protocol: config.get('servers.kibana.protocol'),
+        hostname: config.get('servers.kibana.hostname'),
+        port: config.get('servers.kibana.port'),
+      });
+
+      await page?.goto(kibanaUrl);
+    });
+
+    step('Login to Kibana', async ({ page }) => {
+      const usernameLocator = page.locator('[data-test-subj=loginUsername]');
+      const passwordLocator = page.locator('[data-test-subj=loginPassword]');
+      const submitButtonLocator = page.locator('[data-test-subj=loginSubmit]');
+
+      const noDelayOnUserActions = process.env.TEST_DONT_DELAY_USER_ACTIONS === 'true';
+
+      await usernameLocator?.type('elastic', {
+        delay: noDelayOnUserActions ? 0 : PROFILES.DEFAULT.INPUT_DELAYS.TYPING,
```

Comment:
Hard coding the profiles like this prevents people from choosing a profile via the ENV or something. That's more what I had in mind, being able to run a test as `USER_INPUT_PROFILE=asap` or something.

Reply:
I changed this part to make it dynamic based on the environment but I don't like the idea of constants being in services. Do you have any strong opinions about having it as a service?

Reply:
Well, they're not really "constants", they're the current values based on the configuration for this run in my opinion. I'd prefer we don't start using module scope for things like this as there's no reason for the services to be limited to a single run of the FTR in a single process and the config could come from other services (like the config service maybe) at some point. So, yeah, I guess I kind of do.